tools/lsp/preview.rs

```diff
@@ -109,6 +109,8 @@ async fn reload_preview(preview_component: PreviewComponent) {
 
     let mut builder = slint_interpreter::ComponentCompiler::default();
 
+    builder.set_resource_url_mapper(resource_url_mapper());
```

Comment:
I'd use `#[cfg]`  to have this line only for wasm so no need to build empty function otherwise.